components/base/TabBarPanel.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import classnames from "../../modules/classnames";
+import { GREY_1, GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
+import { BOX_SHADOW_7 } from "../shadowDepths";
+import { ROBOTO } from "../typography";
+
+// Styles
+const Container = styled.div`
+  max-width: auto;
+  border-radius: 5px;
+  background-color: ${WHITE_PRIMARY};
+  :hover {
+    box-shadow: ${BOX_SHADOW_7};
+  }
+`;
+
+const StyledTabBar = styled.div`
+  box-sizing: border-box;
+  border-radius: 5px 5px 0 0;
+  border-color: ${GREY_2};
+  border-style: solid;
+  border-width: 1px 1px 0 1px;
+  background-color: ${GREY_1};
+  display: flex;
+  flex-wrap: nowrap;
+  /* Tab Bar will scroll on smaller screens */
+  overflow-x: auto;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+const TabStyle = styled.div`
```

Comment:
Can we make the cursor a pointer when hovering over an inactive tab?

Reply:
Good catch! Ughh these things that I keep missing!